external_executors/dnanexus_wes/deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Set up dependencies for the lambdas
+wget https://github.com/dnanexus/dxWDL/releases/download/0.61.1/dxWDL-0.61.1.jar
```

Comment:
I'd split this up into `# install dxWDL jar` and `# install dx-toolkit`

Reply:
Okay, will break it up a little and add notes.